client/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const App = () => (
+  <div>
+    let's hang out in san francisco!
+  </div>
+);
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
Index should only have render in it. App component should be imported.

Reply:
Refactored index to only import render method and separated App component to new file.